### ISSUE TITLE
Ignore warnings

### ISSUE THREAD

**Issue by PicoCentauri** [MDAnalysis/mdacli#128]
Currently, MDAnalysis raises a couple of warnings that are unrelated for a user:

```
This module was moved to MDAnalysis.analysis.hydrogenbonds.hbond_autocorrel; hbonds.hbond_autocorrel will be removed in 3.0.0.
Please install the mdahole2 mdakit to use it in MDAnalysis.
More details can be found here: https://www.mdanalysis.org/mdahole2/getting_started.html
Please install the PathSimAnalysis mdakit to use it in MDAnalysis.
More details can be found here: https://www.mdanalysis.org/PathSimAnalysis/getting_started.html
Please install the waterdynamics mdakit to use it in MDAnalysis.
More details can be found here: https://www.mdanalysis.org/waterdynamics/getting_started.html
```

We should suppress them.

**Comment by spyke7** [comment]
yeah these warning always pops up for every command. Working on it!!
